src/ast/mod.rs

```diff
@@ -729,7 +729,7 @@ pub enum Expr {
     /// `[ NOT ] IN (SELECT ...)`
     InSubquery {
         expr: Box<Expr>,
-        subquery: Box<SetExpr>,
+        subquery: Box<Query>,
```

Comment:
🤔  double checking,  this change it is not semver-compatible. I don't think we can release this as 0.56.1, we will have to release it as 0.57.0 as it is a breaking change. 

I will prepare a new branch, and PR